compiler/tests-ocaml/misc/pr7168.ml

```diff
@@ -0,0 +1,82 @@
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in  let x = x+x in  let x = x+x in  let x = x+x in
+  let x = x+x in
+  let _ = f x in
+  ()
+
+let _ =
+  try f 1
+  with Stack_overflow -> Printf.printf "OK\n"
```

Comment:
Stack overflows cannot be caught in Wasm. So, there is no way we can make this example pass.

Reply:
disabled in wasm